src/style.css

```diff
@@ -1 +1,47 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  padding-left: 0;
+  margin: 0;
+  list-style-type: none;
+}
+
+.stars__star {
+  display: flex;
+  background: url(./images/star.svg) center no-repeat;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--0 :nth-child(-n + 0) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--1 :nth-child(-n + 1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n + 2) {
```

Comment:
Avoid using tag names for styling. Instead of using a space before ':nth-child', consider using a class or another selector to target the element. This will improve performance and maintainability.